charts/opentelemetry-collector/values.yaml

```diff
@@ -25,6 +25,7 @@ presets:
   logsCollection:
     enabled: false
     includeCollectorLogs: false
+    storeOffsets: false
```

Comment:
I noticed the filelogreceiver readme used "checkpoints" to describe what the `storage` option does.  Would that be a more user-friendly word instead of `Offsets`?